src/js/britmoji.js

```diff
@@ -31,7 +31,8 @@ const phrases = [
 	"@oomfies",
 	"hi oomfies",
 	createImage("https://media.discordapp.net/stickers/983690705727815731.webp?size=160"),
-	createImage("https://pbs.twimg.com/media/FcJpjQMWQAEI31Z?format=jpg&name=orig")
+	createImage("https://pbs.twimg.com/media/FcJpjQMWQAEI31Z?format=jpg&name=orig"),
+	"fern was here :>"
```

Comment:
move this to the silly little front page theres a list of dot points its so based but also you should put a funny little silly little image here thatd be kinda awesome too

Reply:
Will implement this
Innovation !!!!

Reply:
you are very slow at implementing innovations. this will NOT help britmoji incorporation to be number 1 company in tje world .
move the funny text from phrases to the front page little dot points u know like yea smile meeeeow

many please
valentina (britmoji ceo)

Reply:
the changes have been i-impwemented! innovation can pwoceed uwu!!
i moved my n-name to the funny wist a-and it has beautifuw ~aesthetic~ cowouws.
i a-awso added a uwu k-kawaii picture o-of my kitty to the cwicks ^w^
and guess who added some suuuuper secwet memes to the f-funny funny memes page??!

i can’t wait to b-be t-the 100% number one company in the wowwd!
C- Ciao :3